### PR TITLE
add: add package metadata for cat-master21 projects

### PR DESCRIPTION
**Pull request by cat-master21**

### CODE REVIEW

File: packages/aomp-deb/aomp-deb.pacscript [hunk 1]
```diff
@@ -1,15 +1,10 @@
 name="aomp-deb"
 gives="aomp"
-version="14.0-2"
+version="14.0-3"
+repology="aomp"
```
Comment (wizard-28) [inline packages/aomp-deb/aomp-deb.pacscript:4]: As stated [here](https://github.com/pacstall/pacstall-programs/discussions/845#discussioncomment-2372145) you can't use `pacup` with `aomp` at the moment.

So I guess remove the variable until in the future an equivalent repology package is available

File: packages/atom-editor-deb/atom-editor-deb.pacscript [hunk 2]
```diff
@@ -1,7 +1,8 @@
 name="atom-editor-deb"
 gives="atom"
-version="1.59.0"
+repology="atom"
+version="1.60.0"
 url="https://github.com/$gives/$gives/releases/download/v$version/$gives-amd64.deb"
```
Comment (wizard-28) [inline packages/atom-editor-deb/atom-editor-deb.pacscript:5]:
```suggestion
url="https://github.com/${gives}/${gives}/releases/download/v${version}/${gives}-amd64.deb"
```

File: packages/atom-editor-deb/atom-editor-deb.pacscript [hunk 3]
```diff
@@ -1,7 +1,8 @@
 name="atom-editor-deb"
 gives="atom"
-version="1.59.0"
+repology="atom"
+version="1.60.0"
```
Comment (wizard-28) [inline packages/atom-editor-deb/atom-editor-deb.pacscript:4]: So, `atom` has 2 newest versions in repology,`1.60` (from nix) and `1.60.0` (from others)

https://repology.org/project/atom/versions

So out of the `1.60.0` ones (as this pacscript uses the `x.y.z` version format), which repository do you want to track?

File: packages/balena-etcher-electron-deb/balena-etcher-electron-deb.pacscript [hunk 4]
```diff
@@ -1,6 +1,7 @@
 name="balena-etcher-electron-deb"
 version="1.7.7"
 pkgname="balena-etcher-electron"
+repology="balena-etcher"
```
Comment (wizard-28) [inline packages/balena-etcher-electron-deb/balena-etcher-electron-deb.pacscript:4]:
```suggestion
repology=("project: balena-etcher")
```

File: packages/nyxt-deb/nyxt-deb.pacscript [hunk 5]
```diff
@@ -1,6 +1,7 @@
 name="nyxt-deb"
 version="2.2.4"
 gives="nyxt"
+repology="nyxt"
```
Comment (wizard-28) [inline packages/nyxt-deb/nyxt-deb.pacscript:4]: I guess the AUR (currently has the latest version of the package) tracks pre-releases too, so I guess we would have to track some other repo.

Select a repo from here: https://repology.org/project/nyxt/versions

File: packages/vivaldi-deb/vivaldi-deb.pacscript [hunk 6]
```diff
@@ -1,5 +1,6 @@
 name="vivaldi-deb"
 version="5.1.2567.57-1"
```
Comment (wizard-28) [inline packages/vivaldi-deb/vivaldi-deb.pacscript:2]:
```suggestion
version="5.1.2567.57"
```

File: packages/vivaldi-deb/vivaldi-deb.pacscript [hunk 7]
```diff
@@ -1,5 +1,6 @@
...
+repology="vivaldi"
 url="https://downloads.vivaldi.com/stable/vivaldi-stable_${version}_amd64.deb"
```
Comment (wizard-28) [inline packages/vivaldi-deb/vivaldi-deb.pacscript:4]:
```suggestion
url="https://downloads.vivaldi.com/stable/vivaldi-stable_${version}-1_amd64.deb"
```

File: packages/vivaldi-deb/vivaldi-deb.pacscript [hunk 8]
```diff
@@ -1,5 +1,6 @@
 name="vivaldi-deb"
 version="5.1.2567.57-1"
+repology="vivaldi"
```
Comment (wizard-28) [inline packages/vivaldi-deb/vivaldi-deb.pacscript:3]:
```suggestion
repology=("project: vivaldi")
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/atom-editor-deb/atom-editor-deb.pacscript (hunk 3 above)
Comment (cat-master21) [reply packages/atom-editor-deb/atom-editor-deb.pacscript:4]: I'll be tracking `1.60.0` version as it is most common.

File: packages/nyxt-deb/nyxt-deb.pacscript (hunk 5 above)
Comment (cat-master21) [reply packages/nyxt-deb/nyxt-deb.pacscript:4]: So what should I do? version [3](https://github.com/atlas-engineer/nyxt/releases/tag/3) was a test so after that should be fine. If you want I can temporarily remove `repology`.

Comment (wizard-28) [reply packages/nyxt-deb/nyxt-deb.pacscript:4]: Which repository do you want to track?

File: packages/atom-editor-deb/atom-editor-deb.pacscript (hunk 3 above)
Comment (wizard-28) [reply packages/atom-editor-deb/atom-editor-deb.pacscript:4]: Okay, I've updated Pacup to use the most common version out of the filtrate by default
https://github.com/pacstall/pacup/commit/fec2373f1f1585df0bb94223c75673d2d30ee534

File: packages/atom-editor-deb/atom-editor-deb.pacscript [hunk 9]
```diff
@@ -1,7 +1,8 @@
 name="atom-editor-deb"
 gives="atom"
-version="1.59.0"
-url="https://github.com/$gives/$gives/releases/download/v$version/$gives-amd64.deb"
+repology="atom"
```
Comment (wizard-28) [inline packages/atom-editor-deb/atom-editor-deb.pacscript:3]:
```suggestion
repology=("project: atom")
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/nyxt-deb/nyxt-deb.pacscript (hunk 5 above)
Comment (wizard-28) [reply packages/nyxt-deb/nyxt-deb.pacscript:4]:
```suggestion
repology=("project: nyxt" "repo: void_x86_64")
```

I guess if you're unable to decide Void Linux's repository would be a good choice.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/nyxt-deb/nyxt-deb.pacscript (hunk 5 above)
Comment (cat-master21) [reply packages/nyxt-deb/nyxt-deb.pacscript]: Never knew I could track a specific repo

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**